bench/vibrato-bench/Cargo.toml

```
@@ -6,7 +6,7 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-vibrato = { path = "../../thirdparty/vibrato/vibrato", default-features = false, features = ["unchecked"] }
+vibrato = { path = "../../thirdparty/vibrato/vibrato" }
```

Why use git submodule?
```suggestion
vibrato = "=0.3.0"
```

---

To produce system dictionaries, we employ script files in `thirdparty/vibrato/scripts`. So, we have to install the git submodule. To keep consistency with the produced dictionaries, we specify the dependency to the git submodule.